core/src/main/java/com/kin/ecosystem/core/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -461,6 +465,11 @@ public void deleteAccount(int accountIndex) throws DeleteAccountException {
 		}
 	}
 
+	@Override
+	public ArrayList<String> getWalletsAddress(String kinUserId) {
+		return local.getWalletsAddress(kinUserId);
```

Comment:
Can you get the entire list of wallet addresses for all users on this device?

Reply:
yes